src/BSON/UTCDateTime.c

```diff
@@ -29,6 +29,7 @@
 #include <ext/standard/info.h>
 #include <Zend/zend_interfaces.h>
 #include <ext/date/php_date.h>
+#include <ext/date/lib/timelib.h>
```

Comment:
@derickr: I realize that `ext/date/php_date.h` includes `lib/timelib.h` internally, but I reasoned it was preferable to explicitly include this header if we're going to rely on the `TIMELIB_VERSION` constant it defines.

The only downside to this approach is if there is any risk of the path changing across PHP versions. Do you have any input on that?

Reply:
For the record, Derick confirmed offline that the path is reliable.